<launch>

	
	<arg name="base_frame" 	default="base_link" />
	<arg name="rgbd_frame" 	default="xtion2_link_ee" />
	<arg name="rgb_frame" default="/xtion2_rgb_optical_frame" />
	<arg name="depth_frame" default="/xtion2_depth_optical_frame" />
	<arg name="point_topic"	default="/xtion2/depth/points"/>
	<arg name="depth_topic"	default="/xtion2/depth/image_raw"/>
	<arg name="cut_robot" 	default="false" />

	<node name="xtion2_mask" pkg="cp_clothes_db" type="cp_clothes_db" >
		<param name="base_frame"	value="$(arg base_frame)"/>
		<param name="rgbd_frame" 	value="$(arg rgbd_frame)"/>
		<param name="rgb_frame"		value="$(arg rgb_frame)"/>
		<param name="depth_frame"	value="$(arg depth_frame)"/>
		<param name="point_topic" 	value="$(arg point_topic)"/>
		<param name="depth_topic" 	value="$(arg depth_topic)"/>
		<param name="cut_robot" 	value="$(arg cut_robot)"/>
		<param name="crop_min_z" 	value="0.695"/>
		<param name="nxtion" 	value="2"/>
		<param name="class" 	value="pant"/>
	</node>



</launch>

